run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.CH4

```diff
@@ -398,45 +398,41 @@ VerboseOnCores:              root       # Accepted values: root all
 # NOTES:
 # - These are annual emissions. Seasonality is applied via scale factors
 #   computed from EDGARv6.0 monthly emissions.
+#
+# - Use updated files (v2024-01); these are COARDS-compliant.
 #==============================================================================
 (((EDGARv7
-### Oil ###
-0 EDGAR7_CH4_OIL__1B2a          $ROOT/CH4/v2023-04/EDGARv7/$YYYY/v7.0_FT2021_CH4_$YYYY_PRO_OIL.0.1x0.1.nc          emi_ch4 2010-2021/1-12/1/0 C xy kg/m2/s CH4 32 1 1
-0 EDGAR7_CH4_OTHER__1A1_1B1_1B2 $ROOT/CH4/v2023-04/EDGARv7/$YYYY/v7.0_FT2021_CH4_$YYYY_REF_TRF.0.1x0.1.nc          emi_ch4 2010-2021/1-12/1/0 C xy kg/m2/s CH4 34 1 1
-
-### Gas ###
-0 EDGAR7_CH4_OIL__1B2c          $ROOT/CH4/v2023-04/EDGARv7/$YYYY/v7.0_FT2021_CH4_$YYYY_PRO_GAS.0.1x0.1.nc          emi_ch4 2010-2021/1-12/1/0 C xy kg/m2/s CH4 31 2 1
-
-### Coal ###
-0 EDGAR7_CH4_COAL__1B1a         $ROOT/CH4/v2023-04/EDGARv7/$YYYY/v7.0_FT2021_CH4_$YYYY_PRO_COAL.0.1x0.1.nc         emi_ch4 2010-2021/1-12/1/0 C xy kg/m2/s CH4 30 3 1
+### Oil. gas, coal ###
```

Comment:
Should updating these emissions be in the changelog?

Reply:
@lizziel @msulprizio: This may be a moot point since EDGARv8 will supplant this.

Reply:
Correct, and they're not actually being updated to newer versions just files that are COARDS-compliant.

Reply:
I added a note to CHANGELOG.md in https://github.com/geoschem/geos-chem/pull/1992/commits/e837aba3ec9d112b4ab4cc501707e621e43db042 to indicate files have been updated for COARDS compliance.